whatsagent/api/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
All signup test cases should be moved here.

Reply:
api/tests.py  now has the test cases. Removed the tests.py which was at the same level as views.py